src/util/isFunction.ts

```diff
@@ -0,0 +1,3 @@
+export function isFunction(value: unknown): value is Function {
+  return typeof value === 'function';
```

Comment:
So I looked into this recently and discovered that lodash is doing something way more complicated than`typeof`: https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L3063

Are we okay with replacing it with `typeof` check? I saw some Safari bugs mentioned in there saying that `typeof () => {}` returns `Object`. But I didn't go in the detail.

Reply:
https://github.com/HHongSeungWoo/recharts/blob/0c1aa93cb4ea2de06a31b8efce6a1a27f446af7d/src/util/DataUtils.ts#L19 
How about doing it this way?

Reply:
As https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L3063 indeed seems to handle more edge cases, I believe it is safest to just still use lodash.isfunction